Datadog.AutoInstrumentation.Profiler.sln

```diff
@@ -1,32 +1,32 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30611.23
+# Visual Studio Version 17
+VisualStudioVersion = 17.1.32319.34
```

Comment:
Would it be possible to open it with VS 2019?

Reply:
You'll tell me 😛